templates/packages/order_edit.html

```diff
@@ -0,0 +1,217 @@
+                                </div>
+                                <div class="col-4">
+                                    <input id="destination-city" type="text" class="form-control rounded-0" name="destination_city" list="cities" placeholder="City"/>
+                                </div>
+                                <div class="col-3">
+                                    <select id="destination-area" class="form-select rounded-0" name="destination_area">
+                                        {%for db, human in cities.0.AREA_CHOICES%}
+                                            {%if db == package.origin.city.area%}
+                                                <option>{{human}}</option>
+                                            {%endif%}
+                                        {%endfor%}
+                                        {%for db, human in cities.0.AREA_CHOICES%}
+                                            {%if db != package.origin.city.area%}
+                                                <option>{{human}}</option>
+                                            {%endif%}
+                                        {%endfor%}
+                                    </select>
+                                </div>
+                            </div>
+                            <label for="name" class="form-label text-start">Info</label>
+                            <div id="info" class="row g-0 mb-3">
+                                <div class="col">
+                                    <input id="name" type="text" class="form-control rounded-0" name="full_name" placeholder="Name"/>
+                                </div>
+                                <div class="col">
+                                    <input type="tel" class="form-control rounded-0" name="phone_number" placeholder="Phone Number"/>
+                                </div>
+                            </div>
+                            <label for="notes-text" class="col-form-label">Notes:</label>
+                            <textarea class="form-control rounded-0" id="notes-text" name="notes"></textarea>
+                        </form>
+                    </div>
+                </div>
+                <div class="modal-footer" dir="rtl">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-auto">
+                                <button type="button" class="btn btn-outline-dark" data-bs-dismiss="modal" aria-label="Close">בטל</button>
+                            </div>
+                            <div class="col">
+
+                            </div>
+                            <div class="col-auto">
+                                <button type="button" class="btn btn-outline-success">אשר</button>

```

Comment:
So is this meant to do nothing?

Reply:
Uncommitted changes (at the time) the whole modal is not completely finished. (including the whole work with cookies)